.buildkite/pipeline.yml

```diff
@@ -907,26 +907,66 @@ steps:
       slurm_ntasks: 3
       slurm_gres: "gpu:1"
 
-  - label: "gpu_heldsuarez"
-    key: "gpu_heldsuarez"
+  - label: "gpu_dry_risingbubble"
+    key: "gpu_dry_risingbubble"
     command:
-      - "mpiexec julia --color=yes --project experiments/AtmosGCM/heldsuarez.jl --diagnostics=default --fix-rng-seed"
+      - "mpiexec julia --color=yes --project experiments/TestCase/risingbubble.jl "
+    agents:
+      config: gpu
+      queue: central
+      slurm_ntasks: 1
+      slurm_gres: "gpu:1"
+
+  - label: "gpu_moist_risingbubble"
+    key: "gpu_moist_risingbubble"
+    command:
+            - "mpiexec julia --color=yes --project experiments/TestCase/risingbubble.jl --with-moisture "
```

Comment:
just minor formatting here, YAML is indentation sensitive in a block so it's nice to keep it consistent

Reply:
Thanks!